daprdocs/content/en/concepts/terminology.md

```diff
@@ -20,3 +20,5 @@ This page details all of the common terms you may come across in the Dapr docs.
 | Self-hosted | Windows/macOS/Linux machine(s) where you can run your applications with Dapr. Dapr provides the capability to run on machines in "self-hosted" mode. | [Self-hosted mode]({{< ref self-hosted-overview.md >}})
 | Service | A running application or binary. This can refer to your application or to a Dapr application.
 | Sidecar | A program that runs alongside your application as a separate process or container. | [Sidecar pattern](https://docs.microsoft.com/azure/architecture/patterns/sidecar)
+| Namespacing | Namespacing in Dapr provides isolation, and thus provides multi-tenancy. | Learn more about namespacing [components]({{< ref component-scopes.md >}}), [service invocation]({{< ref service-invocation-namespaces.md >}}), [pub/sub]({{< ref pubsub-scopes.md >}}), and [actors]({{< ref namespaced-actors.md >}})
```

Comment:
Let's add a new section under Concepts here https://docs.dapr.io/concepts/ called "Isolation" and then say that namespacing is used for isolation and multi-tenancy in Dapr. Then provide links to the four articles on namespacing in actors, components, pub/sub topics and service invocation.

At the same time, can we move this https://docs.dapr.io/concepts/service-mesh/ to be a child article of the https://docs.dapr.io/concepts/faq/. It is strange to have this at this level in the concepts folder.

Reply:
I've opened a new issue for this and will update in a separate PR: #4207